### PR TITLE
Refactored Dockerfile

### PR DESCRIPTION
**Pull request by dgonzalezruiz**
This simplifies the syntax and reduces the amount of steps performed.

### CODE REVIEW

**Comment by linki** [comment]
Looks good. But I get this:

```
$ docker build .
...
Step 7/12 : COPY entrypoint.sh gen_config.py factorio.crt /opt
When using COPY with more than one source file, the destination must be a directory and end with a /
```

```
$ docker version
Client:
 Version:      17.03.0-ce
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   60ccb22
 Built:        Thu Feb 23 10:40:59 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.03.0-ce
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   3a232c8
 Built:        Tue Feb 28 07:52:04 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

File: Dockerfile [hunk 1]
```diff
@@ -9,14 +9,11 @@ RUN apt-get update && apt-get dist-upgrade -y && \
 
 WORKDIR /opt
 
-COPY entrypoint.sh gen_config.py factorio.crt /opt/
-COPY factorio_headless_x64_$VERSION.tar.gz /tmp/factorio_headless.tar.gz
+COPY entrypoint.sh gen_config.py factorio.crt /opt
```
Comment (gtaylor) [inline Dockerfile]: As noted, there must be a trailing slash here.

**Review by gtaylor — CHANGES_REQUESTED** [review]
As @linki noted, one change needed before we can merge this.

File: Dockerfile [hunk 2]
```diff
@@ -9,14 +9,11 @@ RUN apt-get update && apt-get dist-upgrade -y && \
```
Comment (HJFinch) [inline Dockerfile:6]: If you also fix the Dockerfile to use apt-get instead of apt for python, I could forget my PR #6

**Review by HJFinch — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (dgonzalezruiz) [reply Dockerfile:6]: Done! Thanks for the suggestion

**Comment by dgonzalezruiz** [comment]
I just updated the PR @gtaylor , do you mind having a look? You can close PR #6 after closing this one.

**Comment by linki** [comment]
I get the following with this:

```console
$ sh make_release.sh 0.15.10
[...]
###
###
# Creating a new map [save.zip]
###
./entrypoint.sh: line 68: /opt/factorio/bin/x64/factorio: No such file or directory
```

File: Dockerfile (hunk 2 above)
Comment (gtaylor) [reply Dockerfile:6]: The `-y` flag should move back after `apt-get` and before `install`.

**Review by gtaylor — CHANGES_REQUESTED** [review]
Just the minor flag nit!

**Comment by gtaylor** [comment]
Also, please squash these commits for me.

**Comment by dgonzalezruiz** [comment]
Closing this pull request to simplify squash

**Pull request closed without merging**